package.json

```diff
@@ -76,6 +77,11 @@
       }
     ],
     "commands": [
+      {
+        "command": "neo3-visual-devtracker.express.install",
+        "title": "Install Neo Express",
```

Comment:
Should this be Install/Update Neo Express?

Reply:
Will update thanks